APMrover2/crash_check.cpp

```diff
@@ -1,9 +1,9 @@
 #include "Rover.h"
 
 // Code to detect a crash or block
-#define CRASH_CHECK_TRIGGER_SEC     2       // 2 seconds blocked indicates a crash
-#define CRASH_CHECK_THROTTLE_MIN    5.0f    // vehicle must have a throttle greater that 5% to be considered crashed
-#define CRASH_CHECK_VEL_MIN         0.08f   // vehicle must have a velocity under 0.08 m/s or rad/s to be considered crashed
+static const uint16_t CRASH_CHECK_TRIGGER_SEC = 2;   // 2 seconds blocked indicates a crash
+static const float CRASH_CHECK_THROTTLE_MIN = 5.0f;  // vehicle must have a throttle greater that 5% to be considered crashed
+static const float CRASH_CHECK_VEL_MIN = 0.08f;      // vehicle must have a velocity under 0.08 m/s or rad/s to be considered crashed
 
```

Comment:
@khancyr I'm pretty sure that with a define it won't use a variable, but directly a literal. Changing defines to variables isn't always worth it, specially for things only used one time - these one-time only numbers are defined just for ease of change (and better readability), otherwise they would be directly in code.